docs/en/02_Creating_pages_and_content/02_Web_Content_Best_Practices.md

```diff
@@ -29,14 +29,20 @@ As a website editor, you should be aware of creating accessible content such as:
 * Ensure there are **no broken links** in your site—again, broken links will make your site rank lower.
 * Provide **text alternatives** for media content, such as ALT text for your images, since search engines can't see images, just their descriptions.
 
+### Site Architecture
+
+The better your site structure, the better your chance of ranking higher in the search engines. An accurate site structure can reduce bounce rate and improve dwell time, both of which will lead to improved rankings. For example Google’s algorithm automatically awards websites with sitelinks Search Engine Results Pages (also known as “SERPs”). And they do so based on great site structure. Silverstripe CMS has built in search-engine-friendly features like hierarchical URLs but site structure will depend mainly on content authors decision-making while creating parent and child pages (site hierarchy).
```

Comment:
```suggestion
An accurate site structure can reduce bounce rate and improve dwell time, both of which will lead to improved search engine rankings. For example Google’s algorithm automatically awards websites with sitelinks Search Engine Results Pages (also known as “SERPs”). And they do so based on great site structure. Silverstripe CMS has built in search-engine-friendly features like hierarchical URLs but site structure will depend mainly on content authors decision-making while creating parent and child pages (site hierarchy).
```

Reply:
```suggestion
The more meaningful and relevant the page names are to the content in your site, the better your chances are of ranking higher in search engines. This can reduce bounce rate and improve dwell time, both of which will lead to improved rankings. For example Google’s algorithm automatically awards websites with sitelinks Search Engine Results Pages (also known as “SERPs”). And they do so based on great site structure. Silverstripe CMS has built in search-engine-friendly features like hierarchical URLs but site structure will depend mainly on content authors decision-making while creating parent and child pages (site hierarchy).
```

Reply:
Thoughts?